examples/grpc_client_streaming.js

```diff
@@ -0,0 +1,72 @@
+import { Client, Stream } from 'k6/x/grpc'
+import { sleep } from 'k6';
+
+let client = new Client();
+client.load([], "./grpc_server/route_guide.proto")
+
+const COORD_FACTOR = 1e7;
+
+// a sample DB of points
+const DB = [
+	{"location":{"latitude":407838351,"longitude":-746143763},"name":"Patriots Path, Mendham, NJ 07945, USA"},
+	{"location":{"latitude":408122808,"longitude":-743999179},"name":"101 New Jersey 10, Whippany, NJ 07981, USA"},
+	{"location":{"latitude":413628156,"longitude":-749015468},"name":"U.S. 6, Shohola, PA 18458, USA"},
+	{"location":{"latitude":419999544,"longitude":-740371136},"name":"5 Conners Road, Kingston, NY 12401, USA"},
+	{"location":{"latitude":414008389,"longitude":-743951297},"name":"Mid Hudson Psychiatric Center, New Hampton, NY 10958, USA"},
+	{"location":{"latitude":419611318,"longitude":-746524769},"name":"287 Flugertown Road, Livingston Manor, NY 12758, USA"},
+	{"location":{"latitude":406109563,"longitude":-742186778},"name":"4001 Tremley Point Road, Linden, NJ 07036, USA"},
+	{"location":{"latitude":416802456,"longitude":-742370183},"name":"352 South Mountain Road, Wallkill, NY 12589, USA"},
+	{"location":{"latitude":412950425,"longitude":-741077389},"name":"Bailey Turn Road, Harriman, NY 10926, USA"},
+	{"location":{"latitude":412144655,"longitude":-743949739},"name":"193-199 Wawayanda Road, Hewitt, NJ 07421, USA"}
+];
+
+// to run this sample, you need to start the grpc server first.
+// to start the grpc server, run the following command in k6 repository's root:
+// go run -mod=mod examples/grpc_server/*.go
```

Comment:
Just FYI, I've reverted this since the proposal won't work. See that we are trying to execute the script relative to the repository root. And if there is a vendor (just in case it should because of our original GRPC tests) running the test without explicitly passing the flag, it will fail.